src/Api/include/Api/Orbit.h

```diff
@@ -0,0 +1,440 @@
+
+#define ORBIT_SCOPE_WITH_COLOR(name, color)
+#define ORBIT_START_WITH_COLOR(name, color)
+#define ORBIT_START_ASYNC_WITH_COLOR(name, id, color)
+#define ORBIT_ASYNC_STRING_WITH_COLOR(str, id, col)
+#define ORBIT_INT_WITH_COLOR(name, value, color)
+#define ORBIT_INT64_WITH_COLOR(name, value, color)
+#define ORBIT_UINT_WITH_COLOR(name, value, color)
+#define ORBIT_UINT64_WITH_COLOR(name, value, color)
+#define ORBIT_FLOAT_WITH_COLOR(name, value, color)
+#define ORBIT_DOUBLE_WITH_COLOR(name, value, color)
+
+#endif  // ORBIT_API_ENABLED
+
+// Material Design Colors #500
+typedef enum {
+  kOrbitColorAuto = 0x00000000,
+  kOrbitColorRed = 0xf44336ff,
+  kOrbitColorPink = 0xe91e63ff,
+  kOrbitColorPurple = 0x9c27b0ff,
+  kOrbitColorDeepPurple = 0x673ab7ff,
+  kOrbitColorIndigo = 0x3f51b5ff,
+  kOrbitColorBlue = 0x2196f3ff,
+  kOrbitColorLightBlue = 0x03a9f4ff,
+  kOrbitColorCyan = 0x00bcd4ff,
+  kOrbitColorTeal = 0x009688ff,
+  kOrbitColorGreen = 0x4caf50ff,
+  kOrbitColorLightGreen = 0x8bc34aff,
+  kOrbitColorLime = 0xcddc39ff,
+  kOrbitColorYellow = 0xffeb3bff,
+  kOrbitColorAmber = 0xffc107ff,
+  kOrbitColorOrange = 0xff9800ff,
+  kOrbitColorDeepOrange = 0xff5722ff,
+  kOrbitColorBrown = 0x795548ff,
+  kOrbitColorGrey = 0x9e9e9eff,
+  kOrbitColorBlueGrey = 0x607d8bff
+} orbit_api_color;
+
+#ifndef __cplusplus
+// The C version of the implementation lives in Orbit.c.
+#define ORBIT_API_INTERNAL_IMPL 1
+#endif
+
+#ifdef ORBIT_API_INTERNAL_IMPL
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+void orbit_api_init();
+void orbit_api_deinit();
+void orbit_api_start(const char* name, orbit_api_color color);
+void orbit_api_stop();
+void orbit_api_start_async(const char* name, uint64_t id, orbit_api_color color);
+void orbit_api_stop_async(uint64_t id);
+void orbit_api_async_string(const char* str, uint64_t id, orbit_api_color color);
+void orbit_api_track_int(const char* name, int value, orbit_api_color color);
+void orbit_api_track_int64(const char* name, int64_t value, orbit_api_color color);
+void orbit_api_track_uint(const char* name, uint32_t value, orbit_api_color color);
+void orbit_api_track_uint64(const char* name, uint64_t value, orbit_api_color color);
+void orbit_api_track_float(const char* name, float value, orbit_api_color color);
+void orbit_api_track_double(const char* name, double value, orbit_api_color color);
+
+#ifdef __cplusplus
+}
+#endif
+
+#else  // ORBIT_API_INTERNAL_IMPL
+
+#if __linux__
+
+#include <dlfcn.h>
+#include <stdio.h>
+
+inline void* orbit_api_get_lib_orbit() {
+  static void* liborbit = dlopen("./liborbit.so", RTLD_LAZY);
+  if (liborbit == nullptr) printf("%s", "ERROR. liborbit.so not found, Orbit API is disabled.\n");
+  return liborbit;
+}
+
+inline void* orbit_api_get_proc_address(const char* name) {
+  static void* liborbit = orbit_api_get_lib_orbit();
+  void* address = liborbit != nullptr ? dlsym(liborbit, name) : nullptr;
+  printf("orbit_api_get_proc_address for %s : %p\n", name, address);
```

Comment:
If I had to choose I would use `stderr` for this kind of logging, I generally feel that we shouldn't interfere with the `stdout` of a program, it could be used for some specific expected output (although this is likely irrelevant for games).

Reply:
Changed it to stderr, it's also more consistent with our logging in general.